ci/azure-travis-template.yaml

```diff
@@ -82,7 +82,7 @@ steps:
   - script: pip install pytest-cov coverage codecov
     displayName: 'Install coverage dependencies'
 - ${{ if eq(parameters.refguide_check, true) }}:
-  - script: pip install matplotlib sphinx==3.1 numpydoc==1.1 "Jinja2<=3.0.3"
+  - script: pip install matplotlib sphinx==3.1 numpydoc "Jinja2<=3.0.3"
```

Comment:
Ark it would have been good to test if we could then remove the pin on Jinja2. This is why I pinged you on the other issue.

Reply:
My bad, I missed this somehow "/. FWIW I tried this locally without the Jinja pin and everything worked.